shared/actions/tracker.js

```diff
@@ -425,7 +425,7 @@ function serverCallMap (dispatch: Dispatch, getState: Function, skipPopups: bool
 
   const requestIdle = f => {
```

Comment:
Sure.. whatever makes sense.. just nice to have some commentary on it cause it feels like it could come back if someone refactors later